tedana/io.py

```diff
@@ -1,245 +1,299 @@
-"""
-=============================
-io module (:mod: `tedana.io`)
-=============================
-
-.. currentmodule:: tedana.io
-
-The io module handles most file input and output in the `tedana` workflow,
-and simplifies some naming function calls with module globals (see "Globals"
-and "Notes" below). Other functions in the module help simplify writing out
-data from multiple echoes or write very complex outputs.
-
-
-Globals
--------
-outdir
-prefix
-convention
-
-
-Naming Functions
-----------------
-set_convention
-set_prefix
-gen_img_name
-gen_json_name
-gen_tsv_name
-add_decomp_prefix
-
-
-File Writing Functions
-----------------------
-write_split_ts
-writefeats
-writeresults
-writeresults_echoes
-filewrite
+"""The io module handles most file input and output in the `tedana` workflow.
 
-
-File Loading Functions
-----------------------
-load_data
-
-
-Helper Functions
-----------------
-new_nii_like
-split_ts
-
-See Also
---------
-`tedana.constants`
-
-
-Notes
------
-The global variables are set by default in the module to guarantee that the
-functions that use them won't fail if a workflow API is not used.
-However, API calls can override the default settings. Additionally, the
-naming functions beginning with "get" all leverage dictionaries defined in
-the `constants.py` module, as the definitions are large.
+Other functions in the module help simplify writing out data from multiple echoes or write very
+complex outputs.
```

Comment:
This description is a bit confusing, particularly what you mean by 'very complex outputs' Consider clarifying.